<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <widget class="QWidget" name="page0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>424</width>
        <height>329</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/mechanism.png</normaloff>:/icons/mechanism.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Mechanism</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="storage_label">
         <property name="text">
          <string>Storage:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="storage_list"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/motor.png</normaloff>:/icons/motor.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Inputs</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="variables_label">
           <property name="text">
            <string>Variables:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="variables_list"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="records_label">
           <property name="text">
            <string>Records:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="records_list"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>259</width>
        <height>121</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/collections.png</normaloff>:/icons/collections.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Collections</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="structures_label">
           <property name="text">
            <string>Structures:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="structures_list"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="triangular_iteration_label">
           <property name="text">
            <string>Triangular iteration:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="triangular_iteration_list"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>119</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/DimensionalSynthesis.png</normaloff>:/icons/DimensionalSynthesis.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Dimensional Synthesis</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="results_label">
         <property name="text">
          <string>Results:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="results_list"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
